flang/include/flang/Evaluate/call.h

```diff
@@ -68,11 +68,13 @@ class ActualArgument {
   private:
     SymbolRef symbol_;
   };
+  using Label = std::uint64_t;
```

Comment:
Why not use `parser::Label`? Or maybe define `Label` for both in `lib/Common/Fortran.h`.

Reply:
Fortran labels are limited to the range 1..99999 if that helps.

Reply:
Because `parser::Label` is an alias so it cannot be pre-declared and I was not willing to include the whole parse-tree.h file here just to get access to this. I've followed your suggestion and defined Label in lib/Common/Fortran.h.

@Peter, I've kept it a uint64_t to use the same type in both place (I think you chose uint64_t in the parse tree to avoid issues before it is validated to be in the correct range). We are not wasting space in ActualArgument since the variant is anyway at least the size than `uint64_t` to accommodate for the expression indirection pointer.